content/docs/use-cases/shared-development-server.md

```diff
@@ -55,14 +55,26 @@ your <abbr>project</abbr>:
 $ dvc config cache.dir /home/shared/dvc-cache
 ```
 
+You could also use a dedicated command to specify the cache directory to use a
+path relative to your project.
+
+```dvc
+$ dvc cache dir /home/shared/dvc-cache
+```
+
+You could choose both of this interchangeably at times but it's more advisable
```

Comment:
I don't think this explanation it correct. Also, for this doc I think it better to keep only the `dvc cache dir` instead of the `dvc config cache.dir`. This PR supposed to be a very simple one - please, let's keep it focused.